src/mock_crust/crust.rs

```diff
@@ -100,6 +101,11 @@ impl<UID: Uid> Service<UID> {
         trace!(target: "crust", "[MOCK] set_service_discovery_listen not implemented in mock");
     }
 
+    /// Return ip address of the peer.
+    pub fn get_peer_ip_addr(&self, _peer: &UID) -> Result<IpAddr, CrustError> {
+        Ok(IpAddr::V4(Ipv4Addr::new(0, 0, 0, 0)))
```

Comment:
This will have problems that peer gets the same value - so writing future tests based on the actual behavior would be difficult. Think it's better in [this PR](https://github.com/maidsafe/routing/pull/1476/files#diff-9d1f471bb4e917b1d0eec91ca3a02ae2R373)